spec/features/merchants/invoices/show_spec.rb

```diff
@@ -32,6 +32,7 @@
 
     it "shows all my items on the invoice including item name, quantity ordered, price item sold for, and invoice item status" do
       visit "/merchants/#{@merchant_1.id}/invoices/#{@invoice_1.id}"
```

Comment:
you may want to add actual items on an invoice instead of the description of the header to have a more robust test. Otherwise its really just testing the view instead of functioning code

Reply:
i concur